SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitClient.java

```diff
@@ -26,11 +26,17 @@ public class SEPADirectDebitClient {
     private final BraintreeClient braintreeClient;
 
     /**
-     * Create a new instance of {@link SEPADirectDebitClient} using a {@link BraintreeClient}.
+     * Initializes a new {@link SEPADirectDebitClient} instance
      *
-     * @param braintreeClient a {@link BraintreeClient}
+     * @param context an Android Context
+     * @param authorization a Tokenization Key or Client Token used to authenticate
      */
-    public SEPADirectDebitClient(@NonNull BraintreeClient braintreeClient) {
+    public SEPADirectDebitClient(@NonNull Context context, @NonNull String authorization) {
+       this(new BraintreeClient(context, authorization));
+    }
+
+    @VisibleForTesting
+    SEPADirectDebitClient(@NonNull BraintreeClient braintreeClient) {
```

Comment:
Are we planning to remove these in a follow-up? Or are we keeping for ease of unit testing?

Reply:
For now they are nice to keep for unit testing since we pass in a `MockBraintreeClient` with responses to various API calls